schemas/calendar-v1.0.0.json

```diff
@@ -0,0 +1,543 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "https://github.com/rayners/fvtt-seasons-and-stars/schemas/calendar-v1.0.0.json",
+  "title": "Seasons & Stars Calendar Schema",
+  "description": "JSON schema for Seasons & Stars calendar format v1.0.0",
+  "type": "object",
+  "required": ["id", "translations", "months", "weekdays"],
+  "additionalProperties": false,
+  "properties": {
+    "id": {
+      "type": "string",
+      "pattern": "^[a-z0-9-]+$",
```

Comment:
This needs to some form of namespacing, like `rayners:calendar-id`, `rayners/calendar-id` or similar.